apps/devhub/views.py

```diff
@@ -227,7 +226,7 @@ def _get_items(action, addons):
                    reviews=(amo.LOG.ADD_REVIEW,))
 
     filter = filters.get(action)
```

Comment:
bonus: rename `filter` to `filter_` if you aren't fed up with all the renaming already :)